ci-operator/step-registry/openshift-qe/control-plane-smoke/openshift-qe-control-plane-smoke-commands.sh

```diff
@@ -22,12 +22,20 @@ set_pods_per_node(){
 }
 
 set_pods_per_node
+echo "######"
+UUID="$(uuidgen)-cpt"
+export UUID
+echo "######"$UUID
 export EXTRA_FLAGS="--pods-per-node=$PODS_PER_NODE --pod-ready-threshold=180000ms --timeout=10m"
 export WORKLOAD=node-density
 ./run.sh
 
 sleep 60;
 export ITERATIONS=1
+echo "######"
+UUID="$(uuidgen)-cpt"
+export UUID
+echo "######"$UUID
```

Comment:
There's no need of printing the UUID, our tests already print them at the beginning

Reply:
Yes, I was doing this for debugging purposes only.

Reply:
Do we want to remove this log now?

Reply:
This is updated, can you ptal?